config/schemas/config.yaml

```diff
@@ -1473,6 +1473,84 @@ properties:
             title: Enable Kafka
             type: boolean
         additionalProperties: false
+      extraRoles:
+        title: Configure extra Roles for application developers
+        description: |
+          Configure extra Roles for application developers
+          The Roles are added to all application developer namespaces configured in user.namespaces
+        type: object
+        properties: {}
+        additionalProperties:
+          properties:
+            rules:
+              title: PolicyRules for this Role
+              type: array
+              default: []
```

Comment:
The items in this list is just the same rules format that is added to roles, right? Could we perhaps link to some kubernetes documentation to make it very clear what to put here?
Same for the other similar things further down.